index.html

```diff
@@ -1,180 +1,371 @@
-    <h1 class="title playfair" >Pritikant Dhal</h1>
-    <p class="my-introduction">
-      Hello! I am a software developer! I can help you build a product, 
-      feature or website. Take a look at my work.<br/><br/>
-      If you like what you see and have a project you need coded,
-      don’t hesitate to contact me.
-    </p>
-      <button class="btn">Start collaboration</button>  
-      <figure class="main-end">
-        <img src="images/icons/image_geometry_2.svg" alt="main section background image">
-      </figure>
-  </section>
-  
-  <!--Project Section-->
-  <section id="projects" class="popins">
-  <h3 id="project-title" class="playfair title">projects</h3>
+  </head>
+  <body>
+    <!-- header section -->
+    <header class="flex">
+      <img src="Images/icons/image_geometry_1.svg" alt="" class="l-t-img" />
+      <nav class="nav">
+        <ul class="nav_links popins">
+          <li><a href="" class="nav_link">Hello</a></li>
+          <li><a href="" class="nav_link">Portfolio</a></li>
+          <li><a href="" class="nav_link">About</a></li>
+          <li><a href="" class="nav_link">Contact</a></li>
+        </ul>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke-width="1.5"
+          stroke="currentColor"
+          class="w-6 h-6"
+          id="menu-1"
+        >
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+          />
+        </svg>
+      </nav>
+    </header>
 
-<article class="project flex">
-  <img src="images/icons/Project-1.svg" alt="Project1 Image1" class="project-img">
-    <h3>Project name goes here</h3>
-   <ul class="project-languages flex">
-      <li class="project-language">HTML/CSS</li>
-      <li class="project-language">Ruby on Rails</li>
-      <li class="project-language">JavaScript</li>
-    </ul>
-  <button class="project-btn flex">
-    <span>See this project</span>
-    <img
-       src="images/icons/ic_arrow_right.svg" width="20px" height="15px" alt="#" srcset="">
-  </button>
-</article>
+    <!-- hero content section -->
+    <sections id="main" class="flex popins">
+      <div class="main-container flex">
+        <h1 class="title playfair">Pritikant Dhal</h1>
+        <p class="my-intro">
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look at my work. <br /><br />
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <button class="btn popins" id="start-collab-btn">
+          start collabration
+        </button>
+      </div>
+      <div class="main-start"></div>
+      <div class="main-end">
+        <img
+          src="Images/icons/image_geometry_2.svg"
+          alt="main section background"
+        />
+      </div>
+    </sections>
```

Comment:
- [ ] Please stick to the Figma design your `hero section` is different from the Figma design so please try to  match your design as much as possible :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-17 01-01-59](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/45226848/aa8e51ed-b498-4d98-9492-4301b81796ba)

**Figma design**
![Screenshot from 2023-06-17 00-46-10](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/45226848/8531f977-d9ed-4335-8e5d-7ec74320755f)


- [ ] If you check your design and Figma design there are lots of spacing issues so kindly fix this issue :+1: 
- [ ] Please use the same `font styling` as mentioned in the Figma design but in your case, your font styling is not matching with Figma design so kindly fix this issue as well :+1: 
- [ ] Please fix your image path as well your image is broken so kindly fix this issue as well :+1:

Reply:
> * [ ]  Please stick to the Figma design your `hero section` is different from the Figma design so please try to  match your design as much as possible +1 check below point_down  the comparison between your design and Figma design slightly_smiling_face
> 
> **Your Design** ![Screenshot from 2023-06-17 01-01-59](https://user-images.githubusercontent.com/45226848/246523531-aa8e51ed-b498-4d98-9492-4301b81796ba.png)
> 
> **Figma design** ![Screenshot from 2023-06-17 00-46-10](https://user-images.githubusercontent.com/45226848/246523556-8531f977-d9ed-4335-8e5d-7ec74320755f.png)
> 
> * [ ]  If you check your design and Figma design there are lots of spacing issues so kindly fix this issue +1
> * [ ]  Please use the same `font styling` as mentioned in the Figma design but in your case, your font styling is not matching with Figma design so kindly fix this issue as well +1
> * [ ]  Please fix your image path as well your image is broken so kindly fix this issue as well +1

@Saritalitu23, this request has not been implemented. Maybe you've changed this in your local machine, but you've not pushed it to your repo.


### Your Design
[your-app-desktop.webm](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/26597432/0529308e-3877-42a9-ba67-1cbdfd012ce3)

### Your Repo Update
![no-commit](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/26597432/75929b12-41ce-4713-94d7-0715bcf57caf)